components/brave_wallet_ui/common/hooks/timeout.ts

```diff
@@ -29,6 +29,8 @@ export default function useTimeout (
       window.addEventListener('mousemove', mousePosition)
       return () => window.removeEventListener('mousemove', mousePosition)
     }
+
+    return () => { }
```

Comment:
@Douglashdaniel We reviewed this hook file and suggested optimizations on a separate issue. However, ts complains here saying Not all code path returns value. Adding a noop would solve that for now.

Reply:
Updated/Removed this hook here https://github.com/brave/brave-core/pull/10943